deploy/helm/charts/templates/kernel-nfs-storageclass.yaml

```
@@ -37,6 +37,19 @@ metadata:
 {{ toYaml .Values.nfsStorageClass.nfsServerResources.limits | indent 10 }}
 {{- end }}
 {{- end }}
+{{- if or .Values.nfsStorageClass.filePermissions.UID .Values.nfsStorageClass.filePermissions.GID .Values.nfsStorageClass.filePermissions.mode }}
```

How about changing this condition as below?  and in values.yaml comment `filePermissions` and check if below suggestion works else I am ok with existing condition also
```suggestion
{{- if .Values.nfsStorageClass.filePermissions }}
```

---

:heavy_check_mark: